TestAssets/TestPackages/dotnet-dependency-context-test/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-preview1-*",
+  "version": "1.0.0-preview2-*",
```

Comment:
Probably the above exception is the reason why this was added in the first place.